src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs

```diff
@@ -1,88 +1,70 @@
 
-        public bool ElementTruncateState { get; set; } = true;
+        [Parameter]
+        public RenderFragment? Actions { get; set; }
 
-        [Parameter] public bool IsMultiline { get; set; } = false;
+        [Parameter]
+        public string? DismissButtonAriaLabel { get; set; }
 
         [Parameter]
-        public MessageBarStyle MessageBarStyle
-        {
-            get => messageBarStyle;
-            set
-            {
-                messageBarStyle = value;
-                ChooseMessageBarIcon();
-                ClassBuilder.Reset();
-            }
-        }
+        public string? OverflowButtonAriaLabel { get; set; }
 
-        protected override string RootElementClass => "bit-msg-bar";
+        [Parameter]
+        public EventCallback OnDismiss { get; set; }
+
+        protected bool HasDismiss { get => (OnDismiss.HasDelegate); }
+
+        protected bool ExpandSingelLine { get; set; }
 
         protected override void RegisterComponentClasses()
         {
-            ClassBuilder.Register(() => MessageBarStyle == MessageBarStyle.Warning ? $"{RootElementClass}-warning-{VisualClassRegistrar()}"
-                                      : MessageBarStyle == MessageBarStyle.Severe ? $"{RootElementClass}-severe-{VisualClassRegistrar()}"
-                                      : MessageBarStyle == MessageBarStyle.Error ? $"{RootElementClass}-error-{VisualClassRegistrar()}"
-                                      : MessageBarStyle == MessageBarStyle.Success ? $"{RootElementClass}-success-{VisualClassRegistrar()}"
-                                      : MessageBarStyle == MessageBarStyle.Blocked ? $"{RootElementClass}-blocked-{VisualClassRegistrar()}"
-                                      : $"{RootElementClass}-default-{VisualClassRegistrar()}");
+            ClassBuilder.Register(() => IsEnabled is false
+                                        ? string.Empty
+                                        : MessageBarType == BitMessageBarType.Info ? $"{RootElementClass}-info-{VisualClassRegistrar()}"
+                                        : MessageBarType == BitMessageBarType.Warning ? $"{RootElementClass}-warning-{VisualClassRegistrar()}"
+                                        : MessageBarType == BitMessageBarType.Error ? $"{RootElementClass}-error-{VisualClassRegistrar()}"
+                                        : MessageBarType == BitMessageBarType.Blocked ? $"{RootElementClass}-blocked-{VisualClassRegistrar()}"
+                                        : MessageBarType == BitMessageBarType.SevereWarning ? $"{RootElementClass}-severe-warning-{VisualClassRegistrar()}"
+                                        : $"{RootElementClass}-success-{VisualClassRegistrar()}");
         }
 
-        public void ToggleElementTruncate()
+        protected void Truncate()
         {
-            ElementTruncateState = !ElementTruncateState;
+            ExpandSingelLine = !ExpandSingelLine;
```

Comment:
based on this line, the method name is not correct.
it should be something like "ToggleExpandSingleLine"

Reply:
Renamed to ToggleExpandSingleLine